### PR TITLE
Add description on quiver head parameters

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

File: lib/matplotlib/quiver.py [hunk 1]
```diff
@@ -201,6 +201,20 @@
 See Also
 --------
 .Axes.quiverkey : Add a key to a quiver plot.
+
+Notes
+-----
+The arrow is drawn as a polygon using the nodes as shown below. The values
+*headwidth*, *headlength* and *headaxislength* are in units of width.
+
+.. image:: /static/quiver_sizes.svg
```
Comment (oscargus) [inline lib/matplotlib/quiver.py:210]:
```suggestion
.. image:: /_static/quiver_sizes.svg
```
(as I am sure you will realize yourself...)

**Comment by oscargus** [comment]
Not sure if it is a rendering issue on GitHub, but it looks like the h in both width and headwidth are overlapping with the line. Moving the texts slightly to the left is probably a good idea if that is also the case in the actual document.

**Comment by timhoffm** [comment]
> it looks like the h in both width and headwidth are overlapping with the line.

Not for me. Let's see how it looks in the doc build.

**Comment by oscargus** [comment]
Not for me either on my Windows machine. The overlap was on my CentOS 7-machine, so probably some lacking font. Looks much nicer on the Windows machine!

**Review by oscargus — APPROVED** [review]

**Pull request merged**